cards.js

```diff
@@ -0,0 +1,137 @@
+    the printing and typesetting industry. Lorem Ipsum has 
+    been the industry's standard dummy text ever since the 
+    1500s, when an unknown printer took a galley of type 
+    and scrambled it to make a type specimen book. It has
+     survived not only five centuries,but also the leap 
+     into electronic typesetting remaining essent`
+  },
+  {
+    id: 2,
+    image: 'images/work2.svg',  
+    title: 'Multi-Post Stories',
+    subtit1: 'CANOPY',
+    subtit2: 'Back End Dev',
+    subtit3: '2015',
+    description: `A daily selection of privately
+    personalized reads; no accounts or
+    sign-ups required.`,
+    lang1_img: 'images/html-tag.png',
+    lang1: 'html',
+    lang2_img: 'images/css-tag.png',
+    lang2: 'css',
+    lang3_img: 'images/js-tag.png',
+    lang3: 'javaScript',
+    popup_description: `Lorem Ipsum is simply dummy text of 
+    the printing and typesetting industry. Lorem Ipsum has 
+    been the industry's standard dummy text ever since the 
+    1500s, when an unknown printer took a galley of type 
+    and scrambled it to make a type specimen book. It has
+     survived not only five centuries,but also the leap 
+     into electronic typesetting remaining essent`
+  },
+  {
+    id: 3,
+    image: 'images/work3.svg',
+    title: 'TONIC',
+    subtit1: 'CANOPY',
+    subtit2: 'Back End Dev',
+    subtit3: '2015',
+    description: `A daily selection of privately
+    personalized reads; no accounts or
+    sign-ups required.`,
+    lang1_img: 'images/html-tag.png',
+    lang1: 'html',
+    lang2_img: 'images/css-tag.png',
+    lang2: 'css',
+    lang3_img: 'images/js-tag.png',
+    lang3: 'javaScript',
+    popup_description: `Lorem Ipsum is simply dummy text of 
+    the printing and typesetting industry. Lorem Ipsum has 
+    been the industry's standard dummy text ever since the 
+    1500s, when an unknown printer took a galley of type 
+    and scrambled it to make a type specimen book. It has
+     survived not only five centuries,but also the leap 
+     into electronic typesetting remaining essent`
+  },
+  {
+    id: 4,
+    image: 'images/work4.svg',  
+    title: 'Multi-Post Stories',
+    subtit1: 'CANOPY',
+    subtit2: 'Back End Dev',
+    subtit3: '2015',
+    description: `A daily selection of privately
+    personalized reads; no accounts or
+    sign-ups required.`,
+    lang1_img: 'images/html-tag.png',
+    lang1: 'html',
+    lang2_img: 'images/css-tag.png',
+    lang2: 'css',
+    lang3_img: 'images/js-tag.png',
+    lang3: 'javaScript',
+    popup_description: `Lorem Ipsum is simply dummy text of 
+    the printing and typesetting industry. Lorem Ipsum has 
+    been the industry's standard dummy text ever since the 
+    1500s, when an unknown printer took a galley of type 
+    and scrambled it to make a type specimen book. It has
+     survived not only five centuries,but also the leap 
+     into electronic typesetting remaining essent`
+  }
+]  
+}
```

Comment:
- [ ] Kindly notice that each object in the array contains at least the following information about the project: name, description, featured image, technologies, link to live version, and link to the project source code.